### ISSUE TITLE
facebookConnectPlugin is not defined

### ISSUE THREAD

**Issue by kaumac** [jeduan/cordova-plugin-facebook4#5]
When calling browserInit method to use facebookConnectPlugin to test in the browser facebookConnectPlugin is undefined. (I'm calling it on deviceReady and only on browser, but it really doesn't work).

There are a few issues about this on the original repo but it seems like no one is supporting it.

**Comment by jeduan** [comment]
@kaumac There are some race conditions. I just published v1.1 which should solve most of them, please take a look.

**Comment by jeduan** [comment]
@kaumac could you resolve?

**Comment by LukeeeeBennett** [comment]
I bumped into this issue today, is there any further advice?

I've noticed it's to do with the fact that when building for the browser platform it's not properly injecting the `<js-module src="www/facebook-browser.js" ...` library as defined in `plugins/cordova-plugin-facebook4/plugin.xml`.

I noticed on android and ios that the network logs successfully request the `www/facebook-native.js` library but on the browser platform there is no attempt to request the `www/facebook-browser.js` library.

I could refactor the library slightly, include it in my frontend dependencies and inject it manually, which would in turn initialize the facebook SDK and allow use of the same API methods for compatibility, but I'd like to avoid the *hackyness*.
